public/static/pathways/her2_pathway.json

```diff
@@ -4,36 +4,43 @@
   "library": "mCODE_Library.cql",
   "criteria": [
     {
+      "id": "1",
       "elementName": "Condition",
       "expected": "Breast Cancer",
       "cql": "\"Primary Cancer Condition\" PrimaryCancer return Tuple{ value: PrimaryCancer.code.text.value, match: PrimaryCancer.code.text.value ~ 'Malignant neoplasm of breast (disorder)' } "
     },
     {
+      "id": "2",
```

Comment:
Also based on the `exportPathway` function below I thought the idea was that these would be deleted in the final exported version

Reply:
Yes it is deleted in the exported version

Reply:
I guess my question was, why do we need to add them in this file then?

Reply:
Ohh, I didn't think was the exported version. I thought these were like the auto saved files. I suppose they can be removed here and added automatically when the pathway is loaded but that seems unnecessary to save the files in a different format from how we need to build them.

Reply:
I was assuming we want the same backend to serve up pathways both to Camino and the builder. There are a couple different options if we do that. We could keep them in the final saved version and just ignore them in Camino. We could delete them on export and re-add them on import. 

If we don't want the same backend to host pathways for both apps, then we can save in one format for the builder then export without the temporary keys.

There are probably more options too. None are perfect so I'm open to suggestions.